src/App.tsx

```diff
@@ -1,61 +1,115 @@
 import './App.scss';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState('');
+  const [isErrorTitle, setIsErrorTitle] = useState(false);
+  const [isErrorUser, setIsErrorUser] = useState(false);
+  const [todos, setTodos] = useState(todosFromServer);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement
+  | HTMLSelectElement>) => {
+    const { name, value } = e.target;
+
+    switch (name) {
+      case 'title':
+        return setTitle(value);
+
+      case 'userId':
+        return setUserId(value);
+
+      default:
+        return 0;
+    }
+  };
+
+  const handleCheckValue = () => {
+    setIsErrorTitle(!title);
+    setIsErrorUser(!userId);
+  };
+
+  const handleAddTodo = (getTitle:string, getUserId:string) => {
+    const lastIdTodo = Math.max(...todos.map(todo => todo.id));
+    const newTodo = {
+      id: lastIdTodo + 1,
+      title: getTitle,
+      completed: false,
+      userId: Number(getUserId),
+    };
+
+    setTodos([...todos, newTodo]);
+  };
+
+  const handleSubmit = (event:React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (title && userId) {
+      handleAddTodo(title, userId);
+    }
+
+    setTitle('');
+    setUserId('');
+  };
 
-export const App = () => {
   return (
     <div className="App">
       <h1>Add todo form</h1>
 
-      <form action="/api/users" method="POST">
+      <form
+        action="/api/users"
+        method="POST"
+        onSubmit={handleSubmit}
+      >
         <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
+          <label>
+            Title:
+            {' '}
```

Comment:
Remove this line. Use template string.

[More info.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals?retiredLocale=uk)

Reply:
Check other similar cases